containers/hawkular-openshift-security-filter/src/main/java/org/hawkular/openshift/auth/TokenAuthenticator.java

```diff
@@ -490,17 +487,18 @@ protected void stringDone(String body) {
                     boolean allowed = allowedNode != null && allowedNode.asBoolean();
                     onRequestResult(serverExchange, connection, allowed);
                 } catch (IOException e) {
-                    onRequestFailure(serverExchange, connection, e);
+                    onRequestFailure(serverExchange, connection, e, true);
                 }
             } else {
                 IOException e = new IOException(StatusCodes.getReason(clientExchange.getResponse().getResponseCode()));
-                onRequestFailure(serverExchange, connection, e);
+                // Retries will not help to solve the issue
+                onRequestFailure(serverExchange, connection, e, false);
```

Comment:
This gets call if the response is anything other than 201. And there are some responses in which we may want to retry (eg 503) and certain ones in which we don't want to retry (eg 400, 401, 403, etc).

I think it makes sense to still retry in some situations which would need to be addressed here.